src/components/Home/Home.component.jsx

```diff
@@ -31,11 +32,10 @@ const Home = () => {
     setLocalToken(token);
 
     //To set the item to the local storage
-
-    localStorage.setItem('token', token);
+    addTokenToStorage(token);
 
     // To go to addItem page
-    history.push('/list', { localToken: token });
+    history.push('/list', { [localToken]: token });
   };
 
```

Comment:
Unsure change origin and source.

Reply:
> I made some suggestions for changes via comments. During testing, I experienced an error when adding a new item after creating a new shopping list. In addition to a flash of the card component when loading the List component of an existing shopping list.

Awesome. Thank you so much! Could you please post the steps to recreate the error/add screenshots? Thank you.

Reply:
> Unsure change origin and source.

Gets rid of the warning about unused `localToken`